src/components/Filter.js

```diff
@@ -0,0 +1,26 @@
+import { useCart } from "../context/cartContext";
+import "../styles/filter.css";
+export default function Filter() {
+  const { filteredData, dispatchData } = useCart();
+  return (
+    <nav className="filter-nav">

```

Comment:
Instead of `nav` you can use `fieldset` or simply a `div` . `nav` should be used if its elements are used for navigation. Let me know what u think !

Reply:
I have changed it to div.
Yes, It's purpose is to provide navigation links